### ISSUE TITLE
I have a question about script generate_superpixel.py

### ISSUE THREAD

**Issue by fakecan** [Baeksweety/superpixel_transformer#2]
On line 95, read the file into PIL.Image.
Is the WSI(svs or tiff) already converted to jpg(or png) before loading?
I don't understand this part very well and would appreciate some clarification.

thx

**Comment by Baeksweety** [comment]
Yes. The input file is jpg format, which is the thumbnail of the pathological images obtained by preprocessing used CLAM.

**Comment by Baeksweety** [comment]
> On line 95, read the file into PIL.Image. Is the WSI(svs or tiff) already converted to jpg(or png) before loading? I don't understand this part very well and would appreciate some clarification.
> 
> thx
